src/effects/StereoToMono.cpp

```diff
@@ -162,6 +162,7 @@ bool EffectStereoToMono::ProcessOne(TrackList &outputs,
    assert(outTrack->IsLeader());
    outTrack->ConvertToSampleFormat(floatSample);
 
+   //VS: could that be removed if we use Mixer::ApplyGain::Mixdown?
```

Comment:
I don't think so. Say pan is 100% to the left, then here `denominator == 1`, but in the case of `ApplyGain::Mixdown`, it would be divided by 2.

Reply:
Thanks, I'll remove that comment